files/build_templates/sonic_debian_extension.j2

```diff
@@ -168,6 +168,10 @@ sudo bash -c "echo enabled=false > $FILESYSTEM_ROOT/etc/sonic/updategraph.conf"
 {% if shutdown_bgp_on_start == "y" %}
 sudo bash -c "echo '{ \"DEVICE_METADATA\": { \"localhost\": { \"default_bgp_status\": \"down\" } } }' >> $FILESYSTEM_ROOT/etc/sonic/init_cfg.json"
 {% endif %}
+{% if enable_pfc_wd_on_start == "y" %}
+sudo bash -c "echo '{ \"DEVICE_METADATA\": { \"localhost\": { \"default_pfc_wd_status\": \"enable\" } } }' >> $FILESYSTEM_ROOT/etc/sonic/init_cfg.json"
```

Comment:
which agent is reading this DEVICE_METADATA default_pfc_wd_status, where is that code?

Reply:
https://github.com/Azure/sonic-utilities/pull/213

Reply:
if the PR is related/depended to other PR, can you mark this information in the PR comments?